mitogen/master.py

```diff
@@ -370,15 +370,19 @@ def _on_forward_log(self, msg):
         if logger is None:
             self._cache[logger_name] = logger = logging.getLogger(logger_name)
 
+        levelno = int(level_s)
         # See logging.Handler.makeRecord()
-        record = logging.LogRecord(
-            name=logger.name,
-            level=int(level_s),
-            pathname='(unknown file)',
-            lineno=0,
-            msg=s,
-            args=(),
-            exc_info=None,
+        record = logging.makeLogRecord(
+            {
+                "name": logger.name,
+                "levelname": logging.getLevelName(levelno),
```

Comment:
Nitpick: the old code didn't explicitly reinterpret the levelno to a level name, but there's a high chance it happened implicitly.

Reply:
It did https://github.com/python/cpython/blob/4556e7e002fb21ddd9e4b16c7266877a4d99ea42/Lib/logging/__init__.py#L229